src/services/confirmation.statement.service.ts

```diff
@@ -37,3 +37,18 @@ export const getConfirmationStatement = async (session: Session, transactionId:
 
   return csSubmissionResource.resource;
 };
+
+export const updateConfirmationStatement = async (session: Session,
```

Comment:
it doesn't look like the returned value from updateConfirmationStatement gets used, I don't think we'd need to return anything from the update so you can remove the return response and the return type.